homework/index.js

```diff
@@ -0,0 +1,30 @@
+const promise = fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(response => response.json());
+
+function selectRepo() {
+  promise.then((data) => {
+    data.forEach((i) => {
+      const repolist = document.createElement('option');
+      const opt = document.createTextNode(i.name);
+      repolist.appendChild(opt);
+      document.getElementById('repoList').appendChild(repolist);
+    });
+  });
+}
+selectRepo();
+
+function myFunction() {
+  promise.then((data) => {
+    data.forEach((i) => {
+      const optionList = document.getElementById('repoList');
```

Comment:
This can be moved outside of the `forEach` scope (along with getting its value in the next line). You could call that `selectedValue` or something.